libnetwork/drivers/overlay/ov_network.go

```diff
@@ -64,7 +64,7 @@ func init() {
 	// Lock main() to the initial thread to exclude the goroutines executing
 	// func setDefaultVLAN() from being scheduled onto that thread. Changes to
 	// the network namespace of the initial thread alter /proc/self/ns/net,
-	// which would break any code which (incorrectly) assumes that that file is
+	// which would break any code which (incorrectly) assumes that the file is
```

Comment:
same with the `that`

Reply:
Hm, yes; looking here, I _guess_ "that file" here refers to `/proc/self/ns/net`? Perhaps we should repeat that here; WDYT?

Reply:
Yeah! I think that makes this sentence clearer.

Reply:
done!